modules/core/test/test_hal_core.cpp

```diff
@@ -46,12 +46,13 @@ enum
 {
     HAL_EXP = 0,
     HAL_LOG = 1,
-    HAL_SQRT = 2
+    HAL_SQRT = 2,
+    HAL_INV_SQRT = 3
 };
 
 TEST(Core_HAL, mathfuncs)
 {
-    for( int hcase = 0; hcase < 6; hcase++ )
+    for( int hcase = 0; hcase < 8; hcase++ )
```

Comment:
I propose to refactor the test to parameterized test. The loop is not needed and GTest names them in logs and highlights failures.